### PR TITLE
[connectInject|syncCatalog] - Add variables for priorityClassName

### PR DESCRIPTION
**Pull request by Lord-Y**
With this PR all my local bats tests are successful:
`1064 tests, 0 failures`

Signed-off-by: Lord-Y <Lord-Y@users.noreply.github.com>

### CODE REVIEW

**Comment by Lord-Y** [comment]
@mitchellh @ishustava can you guys fix the terraform init/plan :)

**Comment by zonArt** [comment]
> @mitchellh @ishustava can you guys fix the terraform init/plan :)

Yes please do so

**Comment by lkysow** [comment]
Hi, @Lord-Y thanks for the PR. Unfortunately neither connect-inject nor sync-catalog support running multiple replicas at this time. For connect-inject there are issues around certificate renewal when running multiple replicas and for sync-catalog two deployments might overwrite one-another's services.

**Comment by Lord-Y** [comment]
@lkysow Thx for the answer about the replicas. I removed this part from the PR to only keep priorityClassName.

File: Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: consul
-version: 0.24.1
+version: 0.25.0
```
Comment (ishustava) [inline Chart.yaml:3]: Could you revert to the current version of the chart? We usually update it ourselves when we release a new version.

**Review by ishustava — COMMENTED** [review]
Hey @Lord-Y,

We've updated the acceptance tests, and they should be skipped now. Sorry about that!  You could also rebase your fork against the latest master so that you don't see these failures anymore.

Everything looks good to me! One comment about the chart version. Once that's fixed, it good to go.

File: Chart.yaml (hunk 1 above)
Comment (Lord-Y) [reply Chart.yaml]: @ishustava Revert done.

**Comment by ishustava** [comment]
Hey @Lord-Y 

thanks for making updates! Somehow, now the GitHub diff is broken and is showing changes that aren't part of your PR. Would you mind fixing that before we merge?

**Comment by Lord-Y** [comment]
Okay I'll close this one an create a clean PR :)

**Pull request closed without merging**